.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--
+
+* Before making a pull request, ensure the changes are for an approved issue.
+* If your changes are not for an approved issue, your pull request can and will be rejected.
+*
+* CHECK https://github.com/PretendoNetwork/REPO_NAME/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved
```

Comment:
Relative link?

Reply:
The link in the checklist is mostly meant for post-rendered use (so the link actually works). Using a relative link is not super clear at a glance, imo. So this comment gives a more visual indication of where to go